mne/label.py

```diff
@@ -1615,9 +1619,14 @@ def grow_labels(subject, seeds, extents, hemis, subjects_dir=None, n_jobs=1,
         labels = _grow_nonoverlapping_labels(subject, seeds, extents, hemis,
                                              vert, dist, names)
 
-    # add a unique color to each label
-    colors = _n_colors(len(labels))
-    for label, color in zip(labels, colors):
+    if colors is None:
+        # add a unique color to each label
+        label_colors = _n_colors(len(labels))
+    else:
+        # use specified colors
+        label_colors = colors
```

Comment:
There is no guarantee or check that `label_colors` is the right size. I think you had the right approach before, namely to do:
```
label_colors = np.empty((len(labels), 3))
label_colors[:] = colors
```
This guarantees that whatever `colors` is, it can broadcast and you will end up with `label_colors` of the right shape.

Reply:
Fine by me, but _n_colors() (label.py) returns array, shape (n, 4).

Reply:
Then you can do `label_colors = np.ones((len(labels), 4)); label_colors[:, :3] = label_colors` if you want. I doubt anyone wants an alpha < 1

Reply:
> 
> 
> Then you can do `label_colors = np.ones((len(labels), 4)); label_colors[:, :3] = label_colors` if you want. I doubt anyone wants an alpha < 1

I'm now allowing four columns and added info about RGB/alpha values to doc-string.